package.json

```diff
@@ -43,6 +43,7 @@
     ]
   },
   "dependencies": {
+    "@akashnetwork/akash-api": "file:akashnetwork-akash-api-0.0.0.tgz",
```

Comment:
this is to be updated when https://github.com/akash-network/akash-api/pull/143 is merged and published to npm